GameData/RP-1/Science/Experiments/Cherenkov.cfg

```diff
@@ -0,0 +1,144 @@
+// Replacing stock experiments
+// ============================================================================
+@PART[*]:HAS[@MODULE[DMModuleScienceAnimateGeneric]:HAS[#experimentID[RP0Cherenkov]]]:FOR[RP-0-Kerbalism]
+{
+	!MODULE[DMModuleScienceAnimateGeneric]:HAS[#experimentID[RP0Cherenkov]] {}
+	MODULE
+	{
+		name = Experiment
+		experiment_id = RP0Cherenkov
+	}
+}
+
+// ============================================================================
+// Reconfigure experiments(avoiding a global nuke to keep people happy)
+// strongly recommendeded to not change anything in this section
+// Separate from above due to Configure Groups.
+// ============================================================================
+@PART[*]:HAS[@MODULE[Experiment]]:AFTER[RP-0-Kerbalism]
+{
+	@MODULE[Experiment]:HAS[#experiment_id[RP0Cherenkov]]
+	{
+		%ec_rate = 0.0005
+		%data_rate = 5 //2 b/s
+		@data_rate /= 7862400 //3 months
+		%requires = OrbitMinEccentricity:0.04
+		%resources = 
+		%experiment_desc = A Cherenkov detector used for the categorization of high-energy particles from solar wind and cosmic rays, particularly gamma rays from beyond the solar system.
+	}
+}
+
+//==========================================//
+//	General Science Module					//
+//==========================================//
+
+@PART[RP0Cherenkov]:FOR[RP-0]
+{	
+	MODULE
+	{
+		name = DMModuleScienceAnimateGeneric
+		experimentID = RP0Cherenkov
+		
+		animationName = needle
+		experimentAnimation = true
+		experimentWaitForAnimation = true
+		keepDeployedMode = 0
+		showEndEvent = false
+		showStartEvent = false
+		showToggleEvent = false
+		startEventGUIName = Open Sensors
+		endEventGUIName = Close Sensors
+		toggleEventGUIName = Toggle Sensors
+		evaDistance = 1.85
+		
+		collectActionName = Retrieve Radiation Data
+		resetActionName = Discard Radiation Data
+		reviewActionName = Review Radiation Data
+		dataIsCollectable = true
+		experimentActionName = Log Radiation
+		rerunnable = true
+		hideUIwhenUnavailable = true
+		useActionGroups = true
+		useStaging = false
+		xmitDataScalar = 1.0
+		usageReqMaskExternal = 4
+		
+		deployingMessage = The radiation sensors are now exposed.
+		waitForAnimationTime = -1
+		resourceExperiment = ElectricCharge
+		resourceExpCost = 0
+		experimentsLimit = 1
+		totalScienceLevel = 1.0
+		asteroidReports = false
+	}
+}
+
+
+//	COSMIC RAY LEVEL 1.5
+//	Cherenkov Detector
+//	Based on the Geiger-Muller Counters used on many spacecraft.
+//	Part: Bluedog Geiger Counter, FASA Geiger Counter
+//====================================================================================
+
+@PART[RP0Cherenkov]:AFTER[RP-0]
+{	
+	@MODULE[DMModuleScienceAnimateGeneric]
+	{
+		@totalScienceLevel = 0.375
+	}
+}
+
+// ============================================================================
+// Add experiment to parts
+// ============================================================================
+@PART[RP0Cherenkov]:BEFORE[RP-0-Kerbalism]

```

Comment:
This block shouldn't be here--in the original cosmic ray file, it's removing cosmic ray 1 from the cosmic ray 2 part, and adding a different module. But in your case you only have 1 tier, so I think you just nuke this whole @PART node. Honestly this might be broken for cosmic ray 2 as well, I gotta check. Honestly this whole thing is kind of a mess, because original ROKerbalism handled the science stuff, but it got merged into RP-1, but when it was merged instead of unifying the patches _both sets of patches_ were left around, which means half this file is adding stuff and the other half is removing it and replacing it with _other_ stuff, when we should just be adding the final result.

Reply:
Checked and cosmic ray 1 and cosmic ray 2 are added differently (!) in RP-1. And the issue you hit is the above, you kept the adding that was done with cosmic ray 1 (depending on first adding the DP module) but _also_ kept something associated with cosmic ray 2 (nuking that module when found). I'm sorry that file is such a mess!

Reply:
Just glad I wasn't the only one confused by that!